backend/src/main/scala/cromwell/backend/package.scala

```diff
@@ -7,16 +7,27 @@ import wdl4s.values.WdlValue
 
 package object backend {
 
+  trait JobKey {
+    def scope: Scope
```

Comment:
I don't know which to change but it seems strange to have a `Job` key contain a `Scope`. For the sake of consistency, could we either call it `ScopeKey` or make it contain a `Call`?